src/main/java/io/weaviate/client6/v1/api/collections/config/ShardStatus.java

```diff
@@ -0,0 +1,10 @@
+package io.weaviate.client6.v1.api.collections.config;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum ShardStatus {
+  @SerializedName("READY")
+  READY,
+  @SerializedName("READONLY")
+  READONLY;
+}
```

Comment:
Am I right to assume these are the only two statuses a shard can be transitioned to by the user? Referencing the [TS client](https://github.com/weaviate/typescript-client/blob/d2c3b42cbca6a2ddb61b73f50e6585c408464977/src/collections/config/index.ts#L75) here.

Reply:
Yeah, that's correct